paasta_tools/api/views/instance.py

```diff
@@ -726,8 +738,13 @@ def instance_status(request):
             )
         elif instance_type == "flink":
             status = flink_instance_status(instance_status, service, instance, verbose)
-            if status is not None:
+            metadata = flink_instance_metadata(instance_status, service, instance, verbose)
+            if status is not None and metadata is not None:
```

Comment:
this can be simplified

    instance_status["flink"] = {}
    if status:
        instance_status["flink"]["status"] = status
    if metadata:
        instance_status["flink"]["metadata"] = metadata